.gitignore

```diff
@@ -79,4 +79,10 @@ target/
 *.mp4
 
 # Virtual Environment
-.venv/
\ No newline at end of file
+.venv*/
+/mu/wheels/*.whl
+/apply-pr.cmd
+/fresh-run.cmd
+/fresh-venv.cmd
+/test.cmd
+/baseline_packages.json
```

Comment:
I guess these were for your internal tools, perhaps they don't need to be in the repo? Or perhaps change to *.cmd or a scripts-subdirectory or whatever, to make it less specific for your setup